db/migrate/20231104023609_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.integer :author_id
+      t.integer :post_id
+      t.text :text
+
+      t.timestamps
+    end
+    add_index :comments, :author_id
+  end
```

Comment:
- [ ] I appreciate the effort you put into working on the comments table. However, I'd like to clarify that the two columns in the table should be referenced from the `post` and `user` tables instead of being passed in as an integer. Below is an example of the expected format:

```ruby
  create_table :comments do |t|
    t.text :text 
    t.references :user, foreign_key: true 
    t.references :post, foreign_key: true
    t.timestamps
  end
```

Please let me know if you have any questions or need further clarification.